nilearn/plotting/js_plotting_utils.py

```diff
@@ -201,6 +201,11 @@ def colorscale(cmap, values, threshold=None, symmetric_cmap=True,
         symmetric_cmap = True
     if symmetric_cmap and vmin is not None:
         warnings.warn('vmin cannot be chosen when cmap is symmetric')
+        vmin = None
+    if threshold is not None:
+        if vmin is not None:
+            warnings.warn('setting both vmin and a threshold is not allowed')
```

Comment:
In the warning add `setting vmin to zero`.
You can use `and` instead of 2 `if` statements.

Reply:
>  You can use `and` instead of 2 `if` statements.

no because vmin is cleared whatever the second if evaluates to

Reply:
Tthat's why I am suggesting this If the user has not set `vmin` to anything they will not expect it have a value. If they have set both `threshold` and `vmin` they'll expect `vmin` to retain that value. When we are warning them that they shouldn't set both values, then we should also make it clear to them that the `vmin` value is being set to zero so there is no confusion to users old and new.